README.md

```diff
@@ -6,6 +6,8 @@
 
 ExDoc is a tool to generate documentation for your Elixir projects. In case you are looking for documentation for Elixir itself, [check out Elixir's website][elixir-lang].
 
+For documentation on writing documentation [check out Elixir's documentation][hex-writing-docs].
```

Comment:
three times the word "documentation" in a sentence is a bit too much I think.
How about:
> To learn more about how to document this project [check out Elixir's documentation][hex-writing-docs]